ims/picasso/rest.py

```diff
@@ -23,7 +23,8 @@ def setup_rpc():
 def start():
     setup_rpc()
     cfg = config.get()
-    app.run(host=cfg.bind_ip, port=cfg.bind_port)
+    app.run(host=cfg.rest_api.ip,
+            port=int(cfg.rest_api.port))
```

Comment:
Would be good to validate this while parsing the config file. That way, we can keep delegate all type validation to the config parser.